### ISSUE TITLE
How to interpret previously failed files?

### ISSUE THREAD

**Issue by abdcef** [brianpetro/obsidian-smart-connections#132]
![image](https://user-images.githubusercontent.com/56148250/228760492-954bed83-8dfb-45b2-ad1c-2551445f3cc3.png)

Does this mean that the embeddings do not contain the content where the files failed? Do I need to check this occasionally and retry failed files anually?

**Comment by brianpetro** [comment]
Yes, if you're being notified, then try running the retry failed files button. 

Not sure what that blank line is about. You could deleted the "failed-embeddings" file to see if that solves it. It will be recreated if any files fail in the future.
